spa/src/components/pageEditor/PageName/PageName.tsx

```diff
@@ -11,6 +12,10 @@ export function PageName() {
   const [editValue, setEditValue] = useState('');
   const fieldRef = useRef<HTMLDivElement>(null);
 
+  const defaultDonationPage =
+    updatedPagePreview?.id === updatedPagePreview?.revenue_program?.default_donation_page &&
+    !!updatedPagePreview?.revenue_program?.default_donation_page;
+
```

Comment:
I think this can be simplfied but I might be missing a nuance. A page should always have a `revenue_program`, and the `revenue_program`'s default page might be `null`, but that's OK since the page ID will never be `null`.

```suggestion
  const defaultDonationPage =
    updatedPagePreview && updatedPagePreview.id === updatedPagePreview.revenue_program.default_donation_page;

```